pages/linux/xdg-settings.md

```diff
@@ -0,0 +1,24 @@
+# xdg-settings
+
+> Get various settings from the desktop environment.
+> More information: <https://portland.freedesktop.org/doc/xdg-settings.html>.
+
+- Print default web browser:
+
+`xdg-settings get default-web-browser`
+
+- Set default web browser to Firefox:
+
+`xdg-settings set default-web-browser firefox.desktop`
+
+- Set default mail URL scheme handler to evolution.desktop:
+
+`xdg-settings set default-url-scheme-handler mailto evolution.desktop`
+
+- Set default PDF document viewer:
+
+`xdg-settings {{pdf-viewer.desktop}}`
```

Comment:
Shouldn't there be a `set`?
```suggestion
`xdg-settings set {{pdf-viewer.desktop}}`
```

Reply:
@adamoudad could you please also apply this suggestion? I've checked and there should indeed be a `set`.